File: pet/TypePets.java
package pet;

public enum TypePets {
    
    AUCUN("", 0, 0, 0 ,0),
    TORTUE("Tortue", 2 , 1, 5 ,25),
    BAT("chauve-souris", 4, 3, 7 ,20);
	
	private String nom;
	private int atk;
	private int vitesse;
    private int defense;
    private float couts;
	
	private TypePets(String nom, int atk, int vitesse, int defense, float couts)
	{
	this.nom = nom;
	this.atk = atk;
	this.vitesse = vitesse;
	this.defense = defense;
    this.couts = couts;
	}
	
	public String getNom()
	{
		return nom;
	}
	public void setNom(String nom)
	{
		this.nom = nom;
	}
    public int getAtk() {
        return atk;
    }
    public void setAtk(int atk) {
        this.atk = atk;
    }
    public int getVitesse() {
        return vitesse;
    }
    public void setVitesse(int vitesse) {
        this.vitesse = vitesse;
    }
    public int getDefense() {
        return defense;
    }
    public void setDefense(int defense) {
        this.defense = defense;
    }
    public float getCouts() {
        return couts;
    }
    public void setCouts(float couts) {
        this.couts = couts;
    }
}

File: Cs/pet/TypePets.cs
using System;

namespace Pet
{
    public class TypePets
    {
        public static readonly TypePets Aucun = new TypePets("", 0, 0, 0, 0);
        public static readonly TypePets Tortue = new TypePets("Tortue", 2, 1, 5, 25);
        public static readonly TypePets Bat = new TypePets("chauve-souris", 4, 3, 7, 20);

        public string Nom { get; private set; }
        public int Atk { get; private set; }
        public int Vitesse { get; private set; }
        public int Defense { get; private set; }
        public float Couts { get; private set; }

        private TypePets(string nom, int atk, int vitesse, int defense, float couts)
        {
            Nom = nom;
            Atk = atk;
            Vitesse = vitesse;
            Defense = defense;
            Couts = couts;
        }
    }
}
